BraintreeCore/src/main/java/com/braintreepayments/api/core/AppLinkCompatibleBrowserUseCase.kt

```diff
@@ -0,0 +1,36 @@
+package com.braintreepayments.api.core
+
+import android.content.pm.PackageManager
+
+class AppLinkCompatibleBrowserUseCase(
+    private val getDefaultBrowserUseCase: GetDefaultBrowserUseCase,
+    private val packageManager: PackageManager
+) {
+
+    operator fun invoke(): Boolean {
+        getDefaultBrowserUseCase(packageManager)?.let {
+            return appLinkCompatibleBrowsers.contains(it)
+        }
+        return false
+    }
+
+    companion object {
+        private val appLinkIncompatibleBrowsers = listOf(
```

Comment:
Do we need to include the list of incompatible browsers if we're only using the list of compatible browsers as an allow list?

Reply:
Yep, I think the best option is to use an allow list instead of a blocklist for the reason you mentioned.